freud/data.py

```diff
@@ -68,9 +68,24 @@ def generate_system(self, num_replicas=1, scale=1, sigma_noise=0, seed=None):
 
         Note:
             Positions are generated in the order of the instance's
-            ``basis_positions``. The first :math:`N_{replica}`
-            positions come from the first basis position, the next
-            :math:`N_{replica}` the second, etc.
+            ``basis_positions``. The first :math:`N_{replica}` positions come
+            from the first basis position, the next :math:`N_{replica}` the
+            second, etc. A NumPy one-liner that generates the list of indices is
+            ``np.repeat(np.arange(len(uc.basis_positions)), np.product(N))``.
```

Comment:
It's not very clear how `N` is defined. It's close to `num_replicas` but if `num_replicas` is a scalar, you may need `num_replicas**3` in 3D and `num_replicas**2` in 2D. That kind of wrecks the "one liner" appeal but it is much more explicit about what is meant.

Reply:
Yeah, I forgot about the scalar case.